### PR TITLE
Getting workload from UI

### PR DESCRIPTION
**Pull request by divya-accolite**

### CODE REVIEW

File: src/main/java/com/envision/Staffing/services/ShiftCalculator.java [hunk 1]
```diff
@@ -85,47 +81,47 @@ private double evaluate(String expression, Clinician[] clinicians, int hour) {
...
 				}
 			}
-			
-		}
-			//System.out.println("SHOULDN'T COME HERE");
-			return 0;
-		}
 
+		}
+		// System.out.println("SHOULDN'T COME HERE");
```
Comment (PayalMantri10) [inline src/main/java/com/envision/Staffing/services/ShiftCalculator.java]: Can remove this println

File: src/main/java/com/envision/Staffing/services/ShiftPlanningService.java [hunk 2]
```diff
@@ -44,37 +46,36 @@ public Output getShiftPlan(Input input) throws IOException {
 		      lowerLimitFactor = input.getLowerLimitFactor(); 
 		}
 		
-		//testing
-		for(int a : shiftPreferences) {
-		System.out.println(a);
-		}
-		
-		System.out.println("lowerLimitFactor" + lowerLimitFactor);
+
```
Comment (PayalMantri10) [inline src/main/java/com/envision/Staffing/services/ShiftPlanningService.java]: Remove this commented code

File: src/main/java/com/envision/Staffing/services/ShiftCalculator.java (hunk 1 above)
Comment (divya-accolite) [reply src/main/java/com/envision/Staffing/services/ShiftCalculator.java]: Will remove it.

File: src/main/java/com/envision/Staffing/services/ShiftPlanningService.java (hunk 2 above)
Comment (divya-accolite) [reply src/main/java/com/envision/Staffing/services/ShiftPlanningService.java]: Yes, Sure

**Pull request merged**